crates/change_json/LICENSE

```diff
@@ -0,0 +1 @@
+../../LICENSE
```

Comment:
Is it mandatory to include such a file in every subproject?

Reply:
Yes, AFAIK...

Reply:
The reason is that the different crates of this monorepo might be published independently to crates.io in the future.